update-86 (#127)
tari-project/tari-dot-com: _updates/2022-08-11-update-86.md

---
layout: update
tag: Developer Update
date: 2022-08-11
author: stringhandler
thumbnail: update-background.jpg
title: New Testnet Esmeralda - We Ate'nt Dead
class: subpage
---

## New testnet launching: Esmeralda

The `dibbler` testnet has been running since January 2022, but there are new features in the development branch that are not in `dibbler`. 
Because adding these features to `dibbler` would have required a development heavy hardfork, and we've already successfully tested hardforks in `dibbler`, 
the development community decided to save time and create a new testnet instead. 

Those features are now significant enough to start testing, and we're proud to annouce a new testnet [Esmeralda](https://en.wikipedia.org/wiki/Granny_Weatherwax). For those of you

> still mining, **Dibbler will continue to run, side-by-side with Esmeralda**. There's still two lambo's up for grabs at the [Tari Testnet Limited store](https://store.tarilabs.com/), which will continue to use `dibbler` tXTR. 

Reviewer: ```suggestion
still mining, **Dibbler will continue to run, side-by-side with Esmeralda**. There are still two Lambos up for grabs in the [Tari Testnet Limited store](https://store.tarilabs.com/), which will continue to use `dibbler` tXTR. 
```

> NOTE: `Esmeralda` testnet will most likely be reset a number of times as we add features, so if you are looking for stability, rather use `dibbler`. 

Some of the new features in `Esmeralda`:

> * [Bulletproofs+](2022-06-24-update-80)

Reviewer: ```suggestion
* [Bulletproofs+](2022-06-24-update-80) - Bulletproofs, but better. Smaller than regular BPs, and with batch verification of range proofs, full-block validation times are something like 60% quicker. 😮 
```

> * [Stealth addresses](2022-08-01-update-84)

Reviewer: ```suggestion
* [Stealth addresses](2022-08-01-update-84) - Tari brought one-sided payments to Mimblewimble. Stealth addresses make the privacy trade-off much smaller. 🕶️ 
```

> * The ability to burn Tari (eventually this will form part of the peg in) [RFC](https://github.com/tari-project/rfcs/pull/10)

Reviewer: ```suggestion
* Tari 🔥  burn 🔥  transactions. Why would you want to do this?  [Find out more](https://github.com/tari-project/rfcs/pull/10)
```
